hooks/useENS.ts

```diff
@@ -0,0 +1,21 @@
+import { ethers } from "ethers";
+import { useEffect, useState } from "react";
+
+const useENS = (address: string) => {
+  const provider = new ethers.providers.JsonRpcProvider(process.env.infura);
```

Comment:
@bigint you can fetch provider from the `Connection` container, similar to how we use a container here:

https://github.com/pickle-finance/pickle-ui/blob/46cc032aac78fdc9730b4711ee0e58781bf6c863/hooks/useGaugeProxy.ts#L11